modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/GeoIpDownloaderTaskExecutor.java

```diff
@@ -144,8 +151,12 @@ private void startTask(Runnable onFailure) {
             GEOIP_DOWNLOADER,
             GEOIP_DOWNLOADER,
             new GeoIpTaskParams(),
-            ActionListener.wrap(r -> {}, e -> {
-                if (e instanceof ResourceAlreadyExistsException == false) {
+            ActionListener.wrap(r -> logger.debug("Started geoip downloader task"), e -> {
+                Throwable t = e;
```

Comment:
Fly-by comment: I believe we can initialize `Throwable t` in one go with a ternary operator
```java
Throwable t = e instanceof RemoteTransportException ? e.getCause() : e;
```

Reply:
Yes, it makes it cleaner, I've changed it